.gitignore

```diff
@@ -13,7 +13,7 @@ swaggerdocs/*
 **.env
 test/cacao/flatfile-db-example.json
 test/routes/__debug_bin2988553005
-
+.idea
```

Comment:
Why was this added?

Reply:
That would be the InteliJ IDEA config folder, like the `.vscode` one for VS Code.